src/main/java/com/uwetrottmann/trakt5/services/Authentication.java

```diff
@@ -39,7 +39,7 @@
      * The {@code user_code} and {@code verification_url} should be presented to the user.
      * @param clientId Application Client Id
      */
-    @POST(TraktV2.OAUTH2_DEVICE_CODE_URL)
+    @POST("oauth/device/code")
```

Comment:
Why the change? Rather keep the public field to not break existing users.

Reply:
I tried to make everything to be in the same style (relative paths for every retrofit interface)